Depends on scalacheck even when scalacheck is not used

I'm running a few tests that do not depend on scalacheck. I've also set things up so that sbt-doctest should not look for dependencies itself:

```scala
doctestWithDependencies := false
doctestTestFramework    := DoctestTestFramework.ScalaTest
```

Running the tests yields the following (truncated) error:

```
Caused by: java.lang.ClassNotFoundException: org.scalacheck.Test$TestCallback
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.scalatest.tools.Framework$ScalaTestTask.loadSuiteClass(Framework.scala:399)
       [...]
```

Adding a scalacheck dependency in `build.sbt` solves the problem.

It's not a huge deal, but it would be better not to use scalacheck classes when scalacheck is not actually used.

If this is something you feel interesting, and you can point me in the right direction, I'd be happy to take a shot at a PR for this.

--- a/src/main/scala/com/github/tkawachi/doctest/ScalaTestGen.scala
+++ b/src/main/scala/com/github/tkawachi/doctest/ScalaTestGen.scala
@@ -17,7 +17,7 @@ object ScalaTestGen extends TestGen {
        |class ${basename}Doctest
        |    extends $st.FunSpec
        |    with $st.Matchers
-       |    with $st.prop.Checkers {
+       |    ${TestGen.withCheckers(parsedList)} {
        |
        |${StringUtil.indent(TestGen.helperMethods, "  ")}
        |

--- a/src/main/scala/com/github/tkawachi/doctest/TestGen.scala
+++ b/src/main/scala/com/github/tkawachi/doctest/TestGen.scala
@@ -21,6 +21,9 @@ object TestGen {
   def importArbitrary(examples: Seq[ParsedDoctest]): String =
     if (containsProperty(examples)) "import org.scalacheck.Arbitrary._" else ""
 
+  def withCheckers(examples: Seq[ParsedDoctest]): String =
+    if (containsProperty(examples)) "with org.scalatest.prop.Checkers" else ""
+
   def containsExample(examples: Seq[ParsedDoctest]): Boolean =
     examples.exists(_.components.exists {
       case _: Example => true

